modules/gapi/samples/pipeline_modeling_tool/utils.hpp

```diff
@@ -91,6 +93,26 @@ typename duration_t::rep timestamp() {
     return duration_cast<duration_t>(now.time_since_epoch()).count();
 }
 
+#define RETHROW_WITH_MSG_IF_FAILED(expr, msg)      \
```

Comment:
Minor comment about naming: it doesn't actually `RE`-throws, suggesting `THROW_WITH_MSG_IF_FAILED`

Reply:
Why not re-throw?

Reply:
I'm ok with the naming